provisioning/security/security-provider/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/SecurityProviderSymmetricKey.java

```diff
@@ -22,6 +22,7 @@
 {
     private static final String HMAC_SHA_256 = "HmacSHA256";
     private final byte[] primaryKey;
+    @SuppressWarnings("FieldCanBeLocal")
```

Comment:
I do not see getter/setter for this field. Is it safe to remove?

Reply:
fwiw the .NET sdk seems to ignore the secondary key as well 😐

Reply:
I would add a getter for this field then. As a user, I'd expect to be able to get both the primary and secondary keys

Reply:
wonder why it's not there in the first place for both dotnet and java.